src/amo/components/Footer/styles.scss

```diff
@@ -3,52 +3,153 @@
 @import "~ui/css/vars";
 
 .Footer {
+  @include font-opensans-regular();
+
   background: $footer-base-color;
   color: $white;
-  padding: 20px;
+  font-weight: lighter;
+  width: 100%;
+
+  a:link,
+  a:visited,
+  a:hover,
+  a:active {
+    @include font-opensans-regular();
+
+    color: $white;
+    font-size: 16px;
+    font-weight: lighter;
+    margin: 0 0 12px;
+    text-decoration: none;
+  }
+
+  a:active,
+  a:hover {
+    color: $white;
+    text-decoration: underline;
+  }
+}
+
+.Footer-wrapper {
+  padding: 58px 20px;
 
   @include respond-to(medium) {
-    display: flex;
-    justify-content: space-between;
-    padding: 20px $header-footer-gutter;
+    padding: 58px $header-footer-gutter;
   }
 
   @include respond-to(large) {
+    display: grid;
+    grid-template-areas:
```

Comment:
Named areas! Nice! I haven't used them yet as stuff's been a bit in-flux, but they are so cool. 👍